src/dist/conf/config.yaml

```diff
@@ -41,7 +41,7 @@ omero.web:
 
 # Configuration for zipkin http tracing
 http-tracing:
-    enabled: false
+    enabled: true
```

Comment:
This will make tracing fail, potentially spectacularly, if there is no Zipkin instance running. Are we sure we want this?

Reply:
we are not including zipkin as part of the standard deployment, so probably better to default to false as in the previous behavior and enable it as needed.

Reply:
Thanks both, set the property back to `false` in [5692aca](https://github.com/glencoesoftware/omero-ms-image-region/pull/123/commits/5692acaeacc96c65c6aa6da8dc6e85ddcc4104cb)

Reply:
An interesting realization that came as I deployied this default configuration in dev instances (where no Zipkin instance is running as far as I can tell) is that setting `http-tracing.enabled=true` via the YAML configuration means the HTTP requests are being sent to the log file. The relevant logic is in https://github.com/glencoesoftware/omero-ms-image-region/blob/1a3435b505466da6460ace49110fa92bbb486e09/src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java#L182-L205 (same in other micro-services) and uses the `LogSpanReporter` from `omero-ms-core`.

I have no strong opinion either way so I will defer to the experience we have built while enabling this functionality. If the current disabled behavior was still found to be too verbose in the logs, we can always adjust the default level in `logback.xml` or add a new configuration value that would prevent these logging entries.